Add files via upload

MrBob238 opened the pull request:
Добавление двух файлов с ДЗ

MrBob238 committed:
Add files via upload

Добавление двух файлов с ДЗ

IgushkinV commented on src/main/Main.java:
```diff
@@ -1 +1,38 @@
+        String[] stringsArray = methods.splitOnSpace(helloWorld); //14. Разбить строку по символу пробел
+                                                                // (т.е. чтобы каждое слово было отдельным элементом массива)
+        String reverseStr = methods.reverse("123456789"); //15. Поменять последовательность символов в строке на обратную.
+
+
+
+
+
+
+    }
+
+
+}
```
Убрать эти пустые строки.

MrBob238 replied:
Пустые строки убрал. Оставались из-за того, что я о них забыл...

IgushkinV commented on src/main/Main.java:
```diff
@@ -1 +1,38 @@
+        String replaceAO = methods.replaceAWithO("Здесь нет символа 'а'."); //12.Заменить все символы "а" в строке на символ "о"
+        String subStr = methods.substringFrom44to90(longString); //13. Получить подстроку с 44 символа по 90 символ
+        //System.out.println(subStr);
+        String[] stringsArray = methods.splitOnSpace(helloWorld); //14. Разбить строку по символу пробел
+                                                                // (т.е. чтобы каждое слово было отдельным элементом массива)
```
Слишком много пустых строк.

MrBob238 replied:
Эти строки были для читаемости длинных комментариев в конце строки.

MrBob238 committed:
Update Main.java

Пустые строки убрал. Закомментированную тоже убрал.
Пустые строки снизу остались, т.к. я забыл про них. Внутри кода пустые строки были - длинный комментарий после оператора шел, в ИДЕ был не читаем, поэтому перенес его вторую часть на новую строку. Не стоит так делать?
